spec/presenters/edition_diff_presenter_spec.rb

```diff
@@ -0,0 +1,140 @@
+require 'rails_helper'
+
+RSpec.describe Presenters::EditionDiffPresenter do
+  subject { described_class }
+  let(:timestamp) { Time.now.utc }
+  let(:content_id) { SecureRandom.uuid }
+
+  let(:change_note) do
+    OpenStruct.new(
+      note: "This was changed",
+    )
+  end
+
+  let(:organisation_links) do
+    [
+      OpenStruct.new(
+        target_content_id: content_id,
+        link_type: "organisations"
+      )
+    ]
+  end
+
+  let(:edition) do
+    double(:edition,
+      analytics_identifier: nil,
+      base_path: "/path",
+      content_store: "draft",
+      description: "A description",
+      details: {},
+      document_type: "answer",
+      first_published_at: timestamp,
+      last_edited_at: timestamp,
+      need_ids: [],
+      phase: "live",
+      public_updated_at: timestamp,
+      publishing_app: "publisher",
+      redirects: [],
+      rendering_app: "government-frontend",
+      routes: [{ "path" => "/path", "type" => "prefix" }],
+      schema_name: "answer",
+      state: "draft",
+      title: "A title",
+      user_facing_version: 1,
+      update_type: "major",
+      links: organisation_links,
+      change_note: change_note
+    )
+  end
+
+  describe "#call" do
```

Comment:
I imagine here it'd be nice to have one test to check core values and then just need to check small sections in the different contexts.

I also wonder if rather than needing to the type out the whole thing you could use as_json on the edition (which thinking about it might be a nice alternative to using `public_send` above